pkg/install/v1alpha1/cstor_pool.go

```diff
@@ -170,6 +171,13 @@ spec:
         app: cstor-pool
         openebs.io/version: {{ .CAST.version }}
         openebs.io/cas-template-name: {{ .CAST.castName }}
+      ownerReferences:
+      - apiVersion: openebs.io/v1alpha1
+        blockOwnerDeletion: true
+        controller: true
+        kind: CStorPool
```

Comment:
How can CSP be an owner when the PR talks of below relationship?
`SPC -> SP -> Pool Deployment -> CSP`

Reply:
Sry the relationship is SPC -> CSP -> Pool Deployment -> SP.